Examples/php/pragmas/runme.php

```diff
@@ -2,4 +2,5 @@
 
 require "example.php";
 
+echo "Version - " . ((new ReflectionExtension('example'))->getVersion());
 ?>
```

Comment:
Hi. I'll be taking it up from now. My exams just finished. Will tie up loose ends and finish this.